### DISCUSSION TITLE
unable to login

### DISCUSSION THREAD

**Discussion by KT-Developer** [Harshdev098/Research-Nexas#32] [Q&A]
after registering as a student and then login in in first time , the below attached error is being displayed on vscode terminal. can someone help on this:
![Screenshot 2024-10-05 120621](https://github.com/user-attachments/assets/da8e7e6d-9c09-4bd3-9cc3-4668f2d1ea25)

**Comment by Harshdev098** [comment]
You can try these approach-
- Just update the MySQL client `npm install mysql@latest`
- Check wheather you have setted a password for your mysql
- Also you can restart the mysql service from terminal after trying above solutions
